specification/cognitiveservices/ContentSafety/models.tsp

```diff
@@ -573,19 +560,20 @@ model AnnotateTextResult {
 @added(Versions.v2023_10_30_Preview)
 @doc("Pre-defined concept.")
 model PreDefinedConcept {
+  @doc("The concept name.")
   concept: string;
+  @doc("The concept description.")
   description: string;
 }
 
 @added(Versions.v2023_10_30_Preview)
 @doc("Label definition.")
-@renamedFrom(Versions.v2024_01_30_Preview, "SubCategory")
-model Class {
+model SubCategory {
```

Comment:
Subcategory is a single word. This is not going to look good in a public-facing API.

Reply:
You even use it as such below in docs. Be consistent. Use single word casing everywhere.

Reply:
Thanks for pointing out! Changed it to Subcategory